### PR TITLE
Update 2019 eCQM Uuid's

### PR DESCRIPTION
**Pull request by saquino0827**
- The ECQM file comes from :  https://ecqi.healthit.gov/system/files/EP_EC_eCQM_2018-05_v2.zip
- The get-strata-uuids-from-ecqm-zip.js is ran as follows: 
    - ./get-strata-and-uuids-from-ecqm-zip.js <path to zip of ecqm>
    - the tool opens the zip in /tmp/ecqm and extracts the uuid's into util/measures/generated-ecqm-data.json
- The purpose of this task was to update the UUIDs for the 2019 measure-data.json file for the vender Health Catalyst to get an early look at these new identifiers.
- The enrich-measures-data.js runs as part of the 2019 build script as per previous years. This script takes the generated json from ./get-strata-and-uuids-from-ecqm-zip.js and adds them into measures-data.json.

### CODE REVIEW

File: scripts/measures/2019/get-strata-uuids-from-ecqm-zip.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env node
+
+/*
```
Comment (sheldon-b) [inline scripts/measures/2019/get-strata-uuids-from-ecqm-zip.js]: This description + usage block is great 👍

**Comment by sheldon-b** [comment]
Not sure if you have Travis access to check on the failures, but they're failing due to a linter failure. If you run `$ npm test` you can see the failure

File: scripts/measures/2019/get-strata-uuids-from-ecqm-zip.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+      denominatorExceptionUuid: 'E76F6606-1DC9-40DE-8A34-5B4B4E859152'
+    }
+  }
+  ...
+*/
+function extractStrata(measure) {
+  // our version of 'strata' are described as 'numerators'
+  // parse out strata descriptions from numerator text
+  // descriptions are like "Numerator 1: Patients who initiated treatment within 14 days of the diagnosis\nNumerator 2: Patients who initiated treatment and who had two or more additional services with an AOD diagnosis within 30 days of the initiation visit"
+  console.log(measure.subjectOf[0].measureAttribute[0].value[0].$.value);
```
[the remark this answers is not in the document]
Comment (saquino0827) [reply scripts/measures/2019/get-strata-uuids-from-ecqm-zip.js]: Removing as it's not actually needed.

**Comment by saquino0827** [comment]
@sheldon-b I have addressed all you comments and this is ready for re-review. Thanks for the input.

**Pull request merged**